src/ImageRunModal.scss

```diff
@@ -18,7 +18,11 @@
 .ct-input-group-spacer-sm.pf-v6-l-flex {
     // Limit width for select entries and inputs in the input groups otherwise they take up the whole space
     > .pf-v6-c-select, .pf-v6-c-form-control:not(.pf-v6-c-select__toggle-typeahead) {
-      max-inline-size: 10rch;
+      max-inline-size: 10ch;
+    }
+
+    #memory-unit-select + .pf-v6-c-form-control__utilities .pf-v6-c-form-control__toggle-icon {
+      color: var(--pf-t--global--icon--color--on-disabled);
```

Comment:
Ideally we have a reference to a PatternFly issue here, is this a known bug? I couldn't reproduce it with their playground.